leds.ato

```diff
@@ -58,18 +58,17 @@ component _KT_0603B from LED:
     signal cathode ~ pin 1
     signal anode ~ pin 2
 
-component _KT_0603G:
+component _KT_0603G from LED:
     # component 0603 Green 509-620mcd
     footprint = "LED0603-RD"
     mpn = "C12624"
-    designator_prefix = "LED"
     v_f = 2.75V
     i_max = 20mA
     # pins
     signal anode ~ pin 2
```

Comment:
@napowderly it seems like the base LED already has the pins specd as well? https://github.com/atopile/generics/pull/6/files#diff-4bf0771413a3a9ce2af3a059fefe07846500059bae511b12101a1e3b8dbfb895R4-R6

Does this make sense? It seems odd to me we'd put them there since some packages might have them the other way around, for example?

Reply:
Yeah, we should remove it.

Reply:
fixed by https://github.com/atopile/generics/pull/9